index.html

```diff
@@ -242,6 +242,46 @@ <h1>About me</h1>
             </ul>
         </div>
     </section>
+    <!--Contact-form-section-->
+    <section class="contact-form">
+        <form action="https://formspree.io/f/xayzobow" method="post" >
+            <h1 id="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+        <fieldset>
+            <label for="name">
+                <input id="name" name="name" type="name" placeholder="Full name" maxlength="30" required />
+            </label>
+        </fieldset>
+        <fieldset>
+            <label for="email">
+                <input id="email" name="email_address" type="email" placeholder="Email address" required />
+            </label>
+        </fieldset>
+            <textarea id="message" name="message_box" type="text" placeholder="Enter text here" maxlength="500" rows="30" cols="10" required ></textarea>
+            <div>
+                <button type="submit" value="Get in touch">Get in touch</button>
+            </div>
+        </form>
+    </section>
+    <hr>
+    <footer>
+            <ul class="logos">
+                <li>
+                    <a class="logo" href=""><img src="icons/github1.svg" alt="Github logo" /></a>
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/linkedin1.svg" alt="Linkedin logo" /></a>
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/angelist1.svg" alt="Angelist logo" /></a>
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/twitter1.svg" alt="Twitter logo" /></a>
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/medium1.svg" alt="Medium logo" /></a>
+                </li>
+            </ul>
+    </footer>
 </body>
 
```

Comment:
- [ ] Good job, @anbehindY 👍 however, just like the previous reviewer mentioned, your form is not consistent with the wireframe yet. I have helped to prepare a more explanatory schedule for comparison:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/220528262-40bb47a4-3e69-4cda-9cd1-0baf8261a3ab.png)|![image](https://user-images.githubusercontent.com/99715640/220528156-f0471628-0873-4471-959b-c119dc8d3417.png)|

- Kindly notice the following:
  - The `border-bottom` line separating the inputs are too dark than the wireframe is.
  - Your `textarea` has physible border lines which are not present in the requirement.
  -  You seem to be missing the last aesthetic design at the the very bottom of the page.
 
Keep the good work, Champ 💯